api/src/main/java/me/lokka30/treasury/api/economy/EconomyProvider.java

```diff
@@ -39,7 +39,7 @@
      *
      * <p>Warning: The Treasury API version is completely different to any other platform version,
      * such as the 'api-version' value in the Bukkit implementation's plugin.yml file.
-     * <p>Warning: Do not use {@link EconomyAPIVersion#getCurrentAPIVersion()}, this is for internal Treasury use only.
+     * <p>Warning: Do not use this method! It is for internal Treasury use only.
```

Comment:
that warning is for economy providers, hence the javadoc is now wrong. Revert it.

Reply:
Wow, thanks for pointing that out. I didn't even have to read your reply to see what was wrong.